assets/src/edit-story/components/header/buttons/index.js

```diff
@@ -47,15 +47,20 @@ const Space = styled.div`
   width: 6px;
 `;
 
+const Spinner = styled.div`
+  position: absolute;
+`;
+
 function Loading() {
   const { isSaving } = useStory((state) => ({
     isSaving: state.state.meta.isSaving,
   }));
   return (
-    <>
-      {isSaving && <CircularProgress size={30} />}
-      <Space />
-    </>
+    isSaving && (
```

Comment:
I actually don't see this spinner for the _Save draft_ button? I see it for the the other buttons correctly (_Publish_, _Update_, _Switch to draft_), but not for _Save draft_. Does that make sense?